packages/@ourworldindata/components/src/Button/Button.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import cx from "classnames"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import { IconDefinition, faArrowRight } from "@fortawesome/free-solid-svg-icons"
+
+type ButtonCommonProps = {
+    text: string
+    className?: string
+    theme: "solid-vermillion" | "outline-vermillion" | "solid-blue"
+    /** Set to null to hide the icon */
+    icon?: IconDefinition | null
+}
+
+type WithHrefProps = {
+    href: string
+    onClick?: never
+}
+
+type WithOnClickProps = {
+    onClick: () => void
+    href?: never
+}
+
+export type ButtonProps =
+    | (ButtonCommonProps & WithHrefProps)
+    | (ButtonCommonProps & WithOnClickProps)
+
+export const Button = ({
+    theme = "solid-vermillion",
+    className,
+    href,
+    onClick,
+    text,
+    icon = faArrowRight,
+}: ButtonProps) => {
+    const classes = cx("owid-btn", `owid-btn--${theme}`, className)
+
+    if (href) {
```

Comment:
Not so sure about this bit,

[Article from MDN explains more fully](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).

Maybe this is something to raise for future work if there is time pressure.